feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/bgp_graceful_restart_test.go

```diff
@@ -331,7 +339,7 @@ func configACL(d *telemetry.Device, name string) *telemetry.Acl_AclSet {
 	tp20 := aclEntry20.GetOrCreateTransport()
 	tp20.SourcePort, _ = tp20.To_Acl_AclSet_AclEntry_Transport_SourcePort_Union(179)
 	a2 := aclEntry20.GetOrCreateIpv4()
-	a2.SourceAddress = ygot.String(ateDst.IPv4CIDR())
+	a2.SourceAddress = ygot.String(ateCIDR)
 	a2.DestinationAddress = ygot.String(aclNullPrefix)
 	a2.Protocol, _ = a.To_Acl_AclSet_AclEntry_Ipv4_Protocol_Union(6)
```

Comment:
Can you parameterize the **6** here using a **const**. It is supposed to be the **IP_TCP** type from : http://ops.openconfig.net/branches/models/master/docs/openconfig-acl.html

Reply:
#332

Reply:
@Ankur19 please could you check if below looks good for the above comment.  

a.Protocol, _ = a.To_Acl_AclSet_AclEntry_Ipv4_Protocol_Union(telemetry.PacketMatchTypes_IP_PROTOCOL_IP_TCP)

Reply:
Lets leave it now! @liulk has already implemented the change, in the other PR.